src/main/java/Arriky.java

```diff
@@ -12,12 +13,21 @@ public static void main(String[] args) {
         while(running) {
             String command = sc.nextLine();
             printSeparation();
-            if(command.equals("bye")) {
+
+            String[] arguments = command.split(" ");
+
+            if (arguments[0].equals("bye")) {
                 endSession();
                 running = false;
-            } else if (command.equals("list")) {
+            } else if (arguments[0].equals("list")) {
                 tl.listTasks();
                 printSeparation();
+            } else if (arguments[0].equals("mark")) {
+                tl.markDone(Integer.parseInt(arguments[1]) - 1);
+                printSeparation();
+            } else if (arguments[0].equals("unmark")) {
+                tl.unmarkDone(Integer.parseInt(arguments[1]) - 1);
+                printSeparation();
             } else {
                 tl.addTask(command);
```

Comment:
(suggestion!) Maybe we can consider using guard clauses/ using ```if``` without the ```else``` blocks to make the happy path more prominent.
see https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html for more details!